docs/software/apple/usage.mdx

```diff
@@ -3,6 +3,71 @@ id: usage
+
+## the App
+
+The bottom app bar has tabs "Messages", "Bluetooth", "Nodes", "Mesh Map" and "Settings".
+
+The __Messages__ tab has channels and direct messages (DMs), comparable to groups and users, where there is a default group, where people can be received you have not known before. Other channels can be configured and there is a channel that will not be visible here, the admin channel. 
+
+__Messages|DirectMessages__ has all nodes as seen in the "Nodes" page, but this time as communication partners, i.e. people and routers and maybe sensors. Makes sense since you might need to give commands to a router or request data from a sensor. The number of people for DMs will grow over time.
+
+The __Bluetooth__ tab has the connection status of your directly connected node. (over BLE) If your node does not show up for connecting, reboot it - it will respond to connect requests for a minute or so. If you want to physically connect to a different node, restart the app and connect here.
+
+__Nodes__ over time builds up a list of nodes your node has "seen". This is technical info about nodes. Do not be surprised by the time that is reported, like "seen in 71 years"; this comes from the node device, not from the Apple app. A proper message would be "seen 24 seconds ago", but some nodes send a wrong timestamp. What is really helpful here is the signal strength (in dbm) and the signal-to-noise ratio (SNR), both of which may be not very precise, but can give an estimation of how far you can send; or if there is an improvement when you use a different antenna (or a different antenna placement or orientation).
+
+The __Mesh Map__ will show all nodes on a map, nicely showing you the area of your reach. Some nodes do not broadcast their location, so you might miss them on the map.
+
+In __Settings__, you can change the configuration of your nodes. Most features are documented here https://meshtastic.org/docs/configuration
+
+In __Settings|AppSettings__, the Name is used when you use the request location feature in a channel or DM. "Block Range Test" will filter out range test messages that you might not want to be shown.
+
+In __Settings|Share Channel QR Code__, a QR code is shown that contains all channels; so after each change in the channel settings, the QR code should be distributed to all nodes you administer.
+
+In __Settings|User__, the "Licensed Operator" switch enables a mode for licensed HAM radio amateurs (may be dependent on the country you live in).
+
+In __Settings|Channels__, channels, which is something like user groups, can be added or deleted. The Channel Role has to be Secondary. The key size should always be 256 bit - all else is too easy to decipher. What you see here is the base64-representation of the PSK. The maximum number of channels in Meshtastic is 8. One can be the admin channel for remote configuration of your nodes. 
```

Comment:
I'd rather link to the channels documentation ( /docs/configuration/radio/channels) here for details and keep this page only about Apple-specifics.